### PR TITLE
Fixes #3553 - Mod notice - lang typo

### LINKED ISSUE #3553: Typo when a thread is waiting for approval
1.8.20 beta

Typo: "and aiting Moderation: 1 unapproved thread."

Narrowed down the error to rogue code on line 783 - global.php
`$moderation_queue = substr_replace($moderation_queue, ' '.$lang->and.' ', strrpos($moderation_queue, $lang->comma), strlen($lang->comma));`

Where do dev's prefer beta testing reports to be posted. Here, or https://community.mybb.com/thread-221648-post-1322856.html ?

### PR DESCRIPTION
**Pull request by Eldenroot**
#3553 

@effone kindly asked me due to his personal reasons to create this PR to fix this issue, all credits go to @effone

### CODE REVIEW

**Comment by Eldenroot** [comment]
https://cdn.discordapp.com/attachments/432869493379694602/539083218410930186/unknown.png

**Comment by Eldenroot** [comment]
Tested, works fine (one or more same type of reports type or different types of reports)

image

**Comment by woostarUK** [comment]
Tested. Works for me.

File: global.php [hunk 1]
```diff
@@ -779,8 +779,9 @@
 
 if(!empty($moderation_queue))
 {
-	$moderation_queue = $lang->sprintf($lang->mod_notice, implode($lang->comma, $moderation_queue));
-	$moderation_queue = substr_replace($moderation_queue, ' '.$lang->and.' ', strrpos($moderation_queue, $lang->comma), strlen($lang->comma));
+	$moderation_queue_last = array_pop($moderation_queue);
+	$moderation_queue = empty($moderation_queue) ? $moderation_queue_last : implode($lang->comma, $moderation_queue).' '.$lang->and.' '.$moderation_queue_last;
```
Comment (dvz) [inline global.php]: Please change the ternary operator to `if` to comply with [1.8 standards](https://docs.mybb.com/1.8/development/standards/).

Comment (effone) [reply global.php]: What about other already existing instances?
snap

Comment (dvz) [reply global.php]: These can be fixed later too for readability (not that important as new code in 1.9 will use PSR standards).

Comment (euantorano) [reply global.php]: Ternary operator removed in 6ec0619.

**Comment by Eldenroot** [comment]
Thank younfpr fixing this PR

**Pull request merged**